acl/src/main/java/org/springframework/security/acls/jdbc/JdbcAclService.java

```diff
@@ -77,9 +78,14 @@
 	// ===================================================================================================
 
 	public JdbcAclService(DataSource dataSource, LookupStrategy lookupStrategy) {
+		this(new JdbcTemplate(dataSource), lookupStrategy);
 		Assert.notNull(dataSource, "DataSource required");
```

Comment:
this assertion is not necessary because JdbcTemplate validates that the dataSource is not null.

Reply:
I'm so sorry! 
[this @Nullable`](https://github.com/spring-projects/spring-framework/blob/master/spring-jdbc/src/main/java/org/springframework/jdbc/support/JdbcAccessor.java#L45) was missleading...
but you're right: the method `afterPropertiesSet` raises an exception, so I've removed the assertion. Sorry for inconvenience...

Reply:
No problem. It is a bit confusing and due the fact there is also a default constructor.

Thanks for the fast turnaround!